.github/workflows/on-merge-to-main.yml

```
@@ -37,7 +37,7 @@ jobs:
     needs: validate_deployment
     steps:
       - name: "Push signed image to ECR, updated SAM template wih image then upload it to the S3 Artifact Bucket"
-        uses: alphagov/di-devplatform-upload-action-ecr@3d5136bf0722e8b11e1c3e8a09f864a4ab767323 # pin@1.1.0
+        uses: govuk-one-login/di-devplatform-upload-action-ecr@3d5136bf0722e8b11e1c3e8a09f864a4ab767323 # pin@1.1.0
```

```suggestion
        uses: govuk-one-login/devplatform-upload-action-ecr@3d5136bf0722e8b11e1c3e8a09f864a4ab767323 # pin@1.1.0
```
We've dropped the `di-` prefix in the new Github org